storage/src/ordinal/storage.rs

```diff
@@ -128,6 +138,14 @@ impl<E: Storage + Metrics + Clock, V: Array> Ordinal<E, V> {
         let mut items = 0;
         let mut intervals = RMap::new();
         for (section, blob) in &blobs {
+            // Skip if bits are provided and the section is not in the bits
+            if let Some(bits) = &bits {
```

Comment:
To minimize memory usage on completed sections, we replace a BitVec of arbitrary length with a single byte in `Metadata`.

We consider a populated index with `None` to be one of these completed ranges.

I think that is pretty jank but the logic "works".